layouts/partials/navbar-version-selector.html

```diff
@@ -0,0 +1,8 @@
+<a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+	{{ .Site.Params.version_menu }}
```

Comment:
This `version_menu` parameter is not used by Kubeflow (and therefore renders blank). Is this a separate "menu only" value that you want set? 

Kubeflow has a `version` value that is used in each repo branch to indicate that site's version. In my testing, if i rename this to `.Site.Params.version`, then it works as i initially expected (ie. i used below, and therefore the main nav bar showed `master` next to the dropdown menu).

My example config.toml

```
version = "master"
githubbranch = "master"

# Add new release versions here
[[params.versions]]
  version = "master"
  githubbranch = "master"
  url = "https://master.kubeflow.org"

[[params.versions]]
  version = "v0.2"
  githubbranch = "v0.2-branch"
  url = "https://v0-2.kubeflow.org"

[[params.versions]]
  version = "v0.3"
  githubbranch = "v0.3-branch"
  url = "https://v0-3.kubeflow.org"

[[params.versions]]
  version = "v0.4"
  githubbranch = "v0.4-branch"
  url = "https://v0-4.kubeflow.org"

[[params.versions]]
  version = "v0.5"
  githubbranch = "v0.5-branch"
  url = "https://v0-5.kubeflow.org"
```

Reply:
True, but I didn't make this change to be backwards compatible with Kubeflow's customizations (even though I borrowed from them), I made it to be backwards compatible with existing Docsy sites - I don't actually think having the current version name as the button/menu name is idea for all cases, especially if you look at some of the older "live" Kubeflow sites (https://v0-4.kubeflow.org/), they all just have a drop down called "master", which may not make it obvious what you can do with the menu. (unless this isn't the intended behaviour?) The equivalent menu in Agones is called Releases.

There's a separate menu name variable called version_menu, which I'll add to the example site config and the docs.

Reply:
We can demonstrate it in the example site but the menu wouldn't point to anything as we do not have any old versions....

Reply:
I've also added a commit for the instructions.

Reply:
Sounds good. Thanks Lisa